### PR TITLE
Format mock numbers

### PR DESCRIPTION
**Pull request by rhofvendahl**
This PR applies appropriate formatting for mock numbers across all schemas, as discussed in https://github.com/w3c-ccg/traceability-vocab/issues/640.

### CODE REVIEW

**Review by TallTed — APPROVED** [review]
I've not looked at every number, but those I have looked at appear to do the right thing.

**Review by OR13 — APPROVED** [review]

File: docs/openapi/components/schemas/common/DCSAShippingInstruction.yml [hunk 1]
```diff
@@ -208,7 +208,7 @@ example: |-
         }
       },
       "email": "xxinau-sales@example.org",
-      "phoneNumber": "+86-555-865-8495"
+      "phoneNumber": "+86-074-555-4172"
```
Comment (mkhraisha) [inline docs/openapi/components/schemas/common/DCSAShippingInstruction.yml:211]: Are we keeping +89? 
I thought we're moving to all +1 with 555 area codes as that is basically guaranteed to be fake.

**Review by mkhraisha — APPROVED** [review]

File: docs/openapi/components/schemas/common/DCSAShippingInstruction.yml (hunk 1 above)
Comment (rhofvendahl) [reply docs/openapi/components/schemas/common/DCSAShippingInstruction.yml:211]: @mkhraisha That's a good question. Changing everything to +1 with 555 area codes would be US-centric but safe. Ideally we could include fake numbers for a variety of country schemes, but this gets tricky as different countries have [different conventions](https://en.wikipedia.org/wiki/Fictitious_telephone_number) for fake numbers - which'd be annoying to implement and keep track of I think.

In this PR I've maintained the format & country codes of non-US numbers, but swapped out the equivalent of area code for 555. My thinking was that this'd be easy to keep track of (e.g. checking whether a fake number is formatted correctly) & make it clear that these numbers should be treated as fake, but depending on the country & its conventions this isn't 100% safe as far as ensuring fake numbers.

I'm open to changing this though, if you think it'd be better to just swap things for +1 with 555 numbers or else take some other approach.

**Review by BenjaminMoe — APPROVED** [review]

**Review by nissimsan — APPROVED** [review]

**Comment by nissimsan** [comment]
@rhofvendahl , please go ahead and merge when the conflict is resolved.

**Pull request merged**